internal/db/schema/migrations/oss/postgres/91/03_indexes_grants_query.up.sql

```diff
@@ -0,0 +1,39 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+  -- For each of these tables, swap the ordering of the
```

Comment:
Do you want to just change the ordering of the where clauses in the grants query instead?

Reply:
I don't think we could change the grants query instead. In most cases the CTEs in the grants query have a where clause on a single column, and are selecting on the other column. Take for example the `user_oidc_managed_groups` cte:

```sql
 user_oidc_managed_groups (id) as (
   select managed_group_id
     from auth_oidc_managed_group_member_account
    where member_id in (select id from user_accounts)
 ),
```

The `where` clause is on `member_id`, and we are selecting `managed_group_id`. The existing index on `auth_oidc_managed_group_member_account` is on `(managed_group_id, member_id)`, which generally won't get used by a query with a where clause that only uses `member_id`. By swapping the index column order to `(member_id, managed_group_id)`, that grants query can now use the index.